### ISSUE TITLE
File > Configure Shortcuts... Shows Alt on Mac

### ISSUE THREAD

**Issue by MrBlockCat** [opentoonz/opentoonz#3130]
### Issue Summary
On Windows, it makes sense shortcuts would show Alt + A for the Control Point Editor Tool, for example. However, on Macs this should show Option + A, as that's what you have to press.

### Screenshot or Video Reference
<img width="500" alt="Screen Shot 2020-03-04 at 8 22 31 AM" src="https://user-images.githubusercontent.com/37186692/75883626-4dae6e00-5df1-11ea-8138-fb738663b000.png">

### Steps to Reproduce
Open File > Configure Shortcuts... on a Mac. Then, set the preset to Adobe Animate(Flash) and open the tools branch of shortcuts, as shown in the screenshot.

### Expected Results
It should show Option + the other key

### Actual Results
It shows Alt + the other key

### System Information
* **OpenToonz Version**: 1.4.0
* **Operating System**: macOS Catalina
* **CPU**: Quad core i5
* **Memory**: 8 GB LPDDR3
* **Graphics Card**: 
* **Graphics Tablet**:

**Comment by MrBlockCat** [comment]
The same also goes for Command on a Mac showing up as Ctrl on screen, even though you press Command to activate the shortcut still.

**Comment by RodneyBaker** [comment]
I can see that being added rather than replace the current one.
Rationale being that the older set is probably for legacy users of Adobe Flash more than current users of Animate.

**Comment by MrBlockCat** [comment]
I agree! Maybe the current one could be renamed “Adobe Flash Pro” and the new one “Adobe Animate” or something like that. Also, I’m working on a somewhat improved version of the new keyboard shortcuts, I’ll put that here tonight hopefully

**Comment by MrBlockCat** [comment]
I just realized my comment and possibly yours was actually to #3131 , whoops haha

**Comment by RodneyBaker** [comment]
That's weird.  I don't even recall opening this issue.
I'll minimize the one's here related to #3131.

**Comment by artisteacher** [comment]
Related to #1327. I think this is a problem with how QT automatically maps Ctrl+Cmd and Alt+Option together. I had hoped that changing the shortcut file itself would help as a workaround, but changing Ctrl to Cmd or Alt to Opt is not recognized.

**Comment by MrBlockCat** [comment]
I think maybe the shortcut save file should stay the same for cross platform compatibility, but when the file's read by OpenToonz to be displayed in this menu or in tooltips ie. in the Toolbar it should show the Mac-specific keys described before
